### DISCUSSION TITLE
Home Assistant 12/12.1

### DISCUSSION THREAD

**Discussion by Stiesel** [davidm-glitch/home-assistant-eheim-digital#25] [Q&A]
A quick enquiry before I unnecessarily open a bug report today: Unfortunately, the addon no longer starts for me since the update.

**Comment by snooze87** [comment]
This is my error message

Setup error, will try again: ‘ClientConnection’ object has no attribute ‘closed’ 

@Stiesel Do you have the same error?

**Comment by Smarthome-Creator** [comment]
@snooze87 same here.

**Comment by davidm-glitch** [comment]
continue in #26
